src/components/endpoints/Assets.vue

```diff
@@ -107,8 +118,8 @@ const create = async () => {
         )
         await deployedContract.methods
           .setData(
-            '0x715f248956de7ce65e94d9d836bfead479f7e70d69b718d47bfe7b00e05b4fe4', //LSP8TokenIdType
-            3
+            ERC725.encodeKeyName('LSP8TokenIdType'), //LSP8TokenIdType
+            `0x${new BN('3').toString('hex', 64)}`
```

Comment:
Isn't that the same as `web3.utils.padLeft(3, 64)`? 

Not sure if we need `BN` library at all, web3 already provides other functions like `web3.utils.numberToHex` etc..

Reply:
Ok, yes, I need to learn all the utils.

Reply:
Inside of the ParamField we do need BN, but everywhere else the utils are fine.

Reply:
Switched to import { padLeft, numberToHex } from "web3-utils" where possible.